src/game/default/g_client_stats.c

```diff
@@ -278,18 +269,14 @@ void G_ClientStats(g_entity_t *ent) {
 
 	// scores
 	client->ps.stats[STAT_SCORES] = 0;
-	if (g_level.intermission_time || client->locals.persistent.show_scores) {
+	if (g_level.intermission_time || client->locals.show_scores) {
 		client->ps.stats[STAT_SCORES] |= 1;
 	}
 
-	if ((g_level.teams || g_level.ctf) && client->locals.persistent.team) { // send team_name
-		if (client->locals.persistent.team == &g_team_good) {
-			client->ps.stats[STAT_TEAM] = CS_TEAM_GOOD;
-		} else {
-			client->ps.stats[STAT_TEAM] = CS_TEAM_EVIL;
-		}
+	if ((g_level.teams || g_level.ctf) && client->locals.persistent.team) { // send team ID, -1 is no team
+		client->ps.stats[STAT_TEAM] = client->locals.persistent.team->id;
 	} else {
-		client->ps.stats[STAT_TEAM] = 0;
+		client->ps.stats[STAT_TEAM] = -1;
```

Comment:
Yeah basically. I used -1 to indicate none, sort of like what pickup item ID does.